src/core/EdFi.DataManagementService.Core/Middleware/ValidateQueryMiddleware.cs

```diff
@@ -78,7 +78,7 @@ out bool totalValue
             }
         }
 
-        // TODO: DMS-312 Validate all other query parameters are valid ones for the current resource
+        // DMS-312 Validate all other query parameters are valid ones for the current resource
```

Comment:
Whoever works on DMS-312 might be looking for this comment, therefore I left it in place though without "TODO".